.github/workflows/bento.yml

```diff
@@ -0,0 +1,13 @@
+name: Bento
```

Comment:
Is this still up to date with the latest Bento Check configs? https://github.com/marketplace/actions/bento-check

Reply:
Right, no, I'll go update the name!